controllers/capabilities/proxyconfigpromote_status_reconciler_test.go

```diff
@@ -39,11 +39,10 @@ func getApiManger() (apimanager *appsv1alpha1.APIManager) {
 		},
 		Spec: appsv1alpha1.APIManagerSpec{
 			APIManagerCommonSpec: appsv1alpha1.APIManagerCommonSpec{
-				AppLabel:                     &appLabel,
-				ImageStreamTagImportInsecure: &trueValue,
```

Comment:
I am wondering if there's anything in deployment that we could use to replace this?
Any thoughts?

Reply:
We decided to remove the `ImageStreamTagImportInsecure` field from the APIManager .spec because Deployments don't expose a way to bypass TLS cert verification out of the box. There are workarounds to skip TLS server cert verification at both the [OpenShift level](https://docs.openshift.com/container-platform/4.14/openshift_images/image-configuration.html#images-configuration-insecure_image-configuration) and the [kubernetes level](https://stackoverflow.com/questions/51893120/kubernetes-pull-from-insecure-docker-registry). We will include these workarounds in docs/release notes.